Dockerfile

```diff
@@ -0,0 +1,75 @@
+FROM ubuntu:16.04
```

Comment:
Why not Alpine or official php image?

Reply:
I was tired of debugging with alpine...Will give next try :)

Reply:
This base image will be on gcloud builder cache, so wont have to pull each time...application dockerfile will be in this form: https://github.com/pagevamp/theme-builder/blob/docker-compose/Dockerfile ...will see alpine though soon...

Reply:
Why on local would someone need to download 175 mb extra :)